src/renderers/native/ReactNativeFiber.js

```diff
@@ -416,4 +417,11 @@ const ReactNative = {
 
 };
 
+if (typeof injectInternals === 'function') {
```

Comment:
What about `onCommitRoot`? It would be a bit weird to make some of methods conditional, but not others.